packages/wdio-cli/src/utils.js

```diff
@@ -196,3 +165,14 @@ export function addServiceDeps(names, packages, update) {
         }
     }
 }
+/**
+ * @todo add JSComments
+ */
+export function convertPackageHashToObject(string, hash = '$--$') {
```

Comment:
do we need this method? How about adding a `short` property to `supportedPackages` or use `name ` for it.